pilot/pkg/bootstrap/server.go

```diff
@@ -924,6 +930,43 @@ func (s *Server) initRegistryEventHandlers() {
 	}
 }
 
+// telemetryHandler trigger to clean AccessLogging cache when telemetry logging configurations changed
+func (s *Server) telemetryHandler(prev config.Config, curr config.Config, event model.Event) {
+	if accessLoggingChanged(prev, curr, event) {
+		// built-in envoy provider related on MeshConfig, so we reused MeshConfigChanged callback
```

Comment:
nit: built-in envoy provider related on MeshConfig -> built-in envoy provider relies on MeshConfig